backend/agent/runner.py

```
@@ -40,11 +62,38 @@ async def _refresh_macro_cache() -> None:
+        btc_raw = market.get("btcPriceRaw", 0.0)
+        eth_raw = market.get("ethPriceRaw", 0.0)
+        if btc_raw > 0 and eth_raw > 0:
+            cutoff = datetime.now(timezone.utc) - timedelta(hours=72)
+            try:
+                with get_db() as db:
+                    db.add(PriceSnapshot(btc_price=btc_raw, eth_price=eth_raw))
+                    db.execute(delete(PriceSnapshot).where(PriceSnapshot.recorded_at < cutoff))
+            except Exception as db_exc:
+                logger.warning("[agent] price_snapshot write failed: %s", db_exc)
+    await broadcast({"market": market})
+    asyncio.create_task(_push_to_cache(build_full_snapshot()))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)